manage_breast_screening/participants/models/symptoms.py

```diff
@@ -0,0 +1,57 @@
+from django.db import models
+
+from ...core.models import BaseModel, ReferenceDataModel
+from .appointment import Appointment
+from .participant import Participant
+
+
+class SymptomType(ReferenceDataModel):
+    LUMP = "lump"
+    SWELLING_OR_SHAPE_CHANGE = "swelling-or-shape-change"
+    SKIN_CHANGE = "skin-change"
+    NIPPLE_CHANGE = "nipple-change"
+    OTHER = "other"
+
+    name = models.CharField(null=False, blank=False)
```

Comment:
Can I see an example of what kind of values you see `name` and `id` fields storing, for a SymptomType record?